scripts/apeldbloader-cloud

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
+#
+# Startup script for cloud APEL cloud
+#
+# chkconfig: 2345 95 05
+# description: APEL dbloader daemon for cloud
```

Comment:
This isn't the daemon itself - just an init script for the daemon. See the comment for `dbld`.